app/src/main/java/com/arygm/quickfix/ui/elements/ScrollableRow.kt

```diff
@@ -106,12 +107,18 @@ fun MainContent() {
 
   val services =
       listOf(
-          Service("Mechanic", R.drawable.mechanic),
-          Service("Gardener", R.drawable.gardener),
-          Service("Electrician", R.drawable.electrician),
-          Service("Painter", R.drawable.painter),
-          Service("Plumber", R.drawable.plumber),
-      )
+          Category(
+              name = "Handyman Services",
+              description = "Mechanic",
+              id = R.drawable.mechanic.toString()),
+          Category(
+              name = "Gardening", description = "Gardener", id = R.drawable.gardener.toString()),
+          Category(
+              name = "Electrical Work",
+              description = "Electrician",
+              id = R.drawable.electrician.toString()),
+          Category(name = "Painting", description = "Paint", id = R.drawable.painter.toString()),
+          Category(name = "Plumbing", description = "Plumber", id = R.drawable.plumber.toString()))
```

Comment:
Why the addition of descriptions (genuinely curious lol)

Reply:
It was just that they were originally done with services, but now we work with categories so that was the only turnaround I could find without refactoring the whole PopularService row. I used the name for actual category name, description for the name on the row for UI and the id for the image id of the popular services.